File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        printf("Height: ");
        height= get_int();
    }
    while(height<0 || height>23);
    for(int i=1; i<=height; i++)
    {
        for(int x=1; x<=height-i; x++)
        {
            printf(" ");
        }
        for(int y=1; y<=i+1; y++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
import cs50
def main():
    while True:
        print("Height: ",end="")
        height = cs50.get_int()
        if height>0 or height<23:
            break
    for i in range(height):
        for j in range(height-i):
            print(" ",end="")
        print("#" * int(i+2),end="")
        print()
        
if __name__ == "__main__":
    main()
    